README.md

```diff
@@ -1 +1,47 @@
-# LinterHello
\ No newline at end of file
+### Hello Microverse
+
+
+### Built With
+
+    HTML
+    CSS
+    Linters
+
+### Live Demo (if available)
+
+    Live Demo Link
+
+### Getting Started
+
+This is an example of how you may give instructions on setting up your project locally. Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.
```

Comment:
- [ ] As the previous reviewer suggested, I kindly suggest that you personalize your README file content and put your project information there because currently only the placeholders from Microverse Template are there as you haven't made any changes to the template.

Reply:
```
Hello @Hamdullah-Andar!

I've just realized that I approved the project instead of requesting for changes.
I am sorry to mislead you. I haven't done that on purpose 😊 
```

Reply:
Hello @topeogunleye 

It's ok, not a problem. :)